### PR TITLE
Update oidc_profile.py

### PR DESCRIPTION
**Pull request by prdp1137**
Fixed the issue of refresh token error when syncing the permissions.

### CODE REVIEW

**Comment by prdp1137** [comment]
This is for the case which involves the "Use refresh tokens for client credentials grant" being turned off by default by handling with try except.
image

Also, @dabocs is this project active?

**Comment by dabocs** [comment]
> This is for the case which involves the "Use refresh tokens for client credentials grant" being turned off by default by handling with try except. image
> 
> Also, @dabocs is this project active?
@pr0d33p yes this is an active project
I will review your PR

**Comment by dabocs** [comment]
@PyKhaled @mazen-mohamed-tamhub

**Review by PyKhaled — APPROVED** [review]

**Review by dabocs — APPROVED** [review]

File: src/django_keycloak/services/oidc_profile.py [hunk 1]
```diff
@@ -248,6 +248,10 @@ def update_tokens(token_model, token_response, initiate_time):
 
     token_model.access_token = token_response['access_token']
     token_model.expires_before = expires_before
+    try:
+        token_model.refresh_token = token_response['refresh_token']
+    except:
+        token_model.refresh_token = None
     token_model.refresh_token = token_response['refresh_token']
```
Comment (dabocs) [inline src/django_keycloak/services/oidc_profile.py:255]: @pr0d33p I think this line needs to be removed

**Comment by dabocs** [comment]
@pr0d33p sorry I approved & merged this PR by mistake, so I reverted it
please create a new one after resolving the comments

**Pull request merged**